modules/customize-certificates-add-service-serving-validating-webhook.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-validating-webhook_{context}"]
+= Add the service CA bundle to a ValidatingWebhookConfiguration
+
+You can annotate a ValidatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+.Procedure
+
+. Annotate the ValidatingWebhookConfiguration with `service.beta.openshift.io/inject-cabundle=true`:
++
+----
+$ oc annotate validatingwebhookconfigurations <validatingwebhook-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<validatingwebhook-name>` with the name of the ValidatingWebhookConfiguration to annotate.
++
+For example, use the following command to annotate the ValidatingWebhookConfiguration `foo`:
++
+----
+$ oc annotate validatingwebhookconfigurations foo service.beta.openshift.io/inject-cabundle=true
+----
+
+. View the ValidatingWebhookConfiguration to ensure that the certificate has been generated:
++
+----
+$ oc get validatingwebhookconfigurations <validatingwebhook-name> -o yaml
+----
++
+It is displayed as `caBundle` in the YAML output:
```

Comment:
The CA bundle will appear in the `clientConfig.caBundle` field of all webhooks in the YAML output.